torch/csrc/jit/serialization/export.cpp

```diff
@@ -915,29 +916,33 @@ void check_onnx_proto(const std::string& proto_string) {
     onnx::checker::check_model(model);
 }
 
+void moduleMethodsTuple(
```

Comment:
Is there a header file in common between `import.cpp` and `export_module.cpp` that we could put this declaration in?

Reply:
The common header is fine, but may be unnecessarily included in other files. Any disadvantage of declaration?

Reply:
Only that someone changing the signature of the implementation would need to know to update this declaration - I'd probably assume that if a function were used publicly, there would be a declaration in a header somewhere. But doing it this way might be more of a convention in this codebase. Are you following an existing idiom? I'm fine with it if so.